lib/hooks/use-pinheader.tsx

```diff
@@ -0,0 +1,15 @@
+import type { PinHeaderProps } from "@tscircuit/props"
+import { createUseComponent } from "./create-use-component"
+
+export const usePinHeader = (
+  name: string,
+  options: {
+    pinLabels: PinHeaderProps["pinLabels"]
+  },
```

Comment:
Looking at your code vs other examples like useChip, i think “options” should actually be props: PinHeaderPrips